### ISSUE TITLE
accessibility issue 34 (medium) - put pagination buttoms at botton in nav element

### ISSUE THREAD

**Issue by yulgit1** [ycba-cia/blacklight-collections2#249]
The pagination buttons should be within a <nav> landmark with an aria-label of “pagination”.

**Comment by yulgit1** [comment]
addressed: https://github.com/ycba-cia/blacklight-collections2/commit/81441eda6e41c301db6e6767cdc74696cf473e06
